packages/jest-haste-map/src/index.ts

```diff
@@ -665,16 +668,17 @@ class HasteMap extends EventEmitter {
   /**
    * Creates workers or parses files and extracts metadata in-process.
    */
-  _getWorker(options: ?{forceInBand: boolean}): WorkerInterface {
+  _getWorker(options?: {forceInBand: boolean} | null): WorkerInterface {
```

Comment:
I wonder if we could get rid of that null

Reply:
probably